docs/reference/esql/functions/grouping-functions.asciidoc

```diff
@@ -0,0 +1,14 @@
+[[esql-group-functions]]
+==== {esql} aggregate functions
+
+++++
+<titleabbrev>Grouping functions</titleabbrev>
+++++
+
+The <<esql-stats-by>> command supports these grouping functions:
```

Comment:
Maybe we should add a short explanation?

E.g. "They are intended for use in the `BY` - clause, but can also be used in the aggregation expressions" (provided this is actually accurate :D )

Reply:
Good point -- I've added an explanation with a BUCKET example. If we'll add another function, we can generalise. We don't say much for aggregate functions either (where one can use them and how), so I guess examples help better?